resources/views/vote/show.blade.php

```diff
@@ -110,7 +110,7 @@
                 intersect: false
             },
             scales: {
-                yAxes: [{
+                y: [{
```

Comment:
I don't know the JS library, but I was just wondering... In `reports.trainings` the scales are made with `x` and `y` with pure JS objects, not wrapped in an array. Is it intentional that it's still wrapped in an array here?

Reply:
Good catch, solved in push. Also forgot to init charts on this view after the change to charts.js v3 in previous commits.